integrations/access/common/config.go

```diff
@@ -35,12 +35,16 @@ type PluginConfiguration interface {
 	GetTeleportClient(ctx context.Context) (teleport.Client, error)
 	GetRecipients() RawRecipientsMap
 	NewBot(clusterName string, webProxyAddr string) (MessagingBot, error)
+	GetUsersAsRecipients() bool
 }
 
 type BaseConfig struct {
 	Teleport   lib.TeleportConfig
 	Recipients RawRecipientsMap `toml:"role_to_recipients"`
 	Log        logger.Config
+	// UsersAsRecipients is set for bots where the individual assigned
+	// reviewers on the access requests should be considered valid recipients.
+	UsersAsRecipients bool
```

Comment:
I'm not sure I understand this tbh. Could you give an example of why this is different from existing plugins?

Reply:
For the slack plugin the the recipients of the access request  get passed to get message recipients which does not make sense for the opsgenie plugin. This is to let onPendingRequest know not to call that and instead let the plugin figure it out itself.

Reply:
But why do we need this additional flag on the base config? Can we just override `GetRecipients()` on the Opsgenie config method and have it return appropriate recipients?